diff --git a/format/private/format.sh b/format/private/format.sh
--- a/format/private/format.sh
+++ b/format/private/format.sh
@@ -137,6 +137,8 @@
       'Markdown') patterns=('contents.lr' '*.md' '*.livemd' '*.markdown' '*.mdown' '*.mdwn' '*.mkd' '*.mkdn' '*.mkdown' '*.ronn' '*.scd' '*.workbook') ;;
       'Protocol Buffer') patterns=('*.proto') ;;
       'Python') patterns=('.gclient' 'DEPS' 'SConscript' 'SConstruct' 'wscript' '*.py' '*.cgi' '*.fcgi' '*.gyp' '*.gypi' '*.lmi' '*.py3' '*.pyde' '*.pyi' '*.pyp' '*.pyt' '*.pyw' '*.rpy' '*.spec' '*.tac' '*.wsgi' '*.xpy') ;;
+      # Added TRLC support
+      'TRLC') patterns=('*.trlc' '*.rsl') ;;
       'Rust') patterns=('*.rs' '*.rs.in') ;;
       'SQL') patterns=('*.sql' '*.cql' '*.ddl' '*.inc' '*.mysql' '*.prc' '*.tab' '*.udf' '*.viw') ;;
       'SCSS') patterns=('*.scss') ;;
diff --git a/format/private/formatter_binary.bzl b/format/private/formatter_binary.bzl
--- a/format/private/formatter_binary.bzl
+++ b/format/private/formatter_binary.bzl
@@ -8,6 +8,7 @@
     "Markdown": "prettier",
     # NB: includes LESS and SASS
     "CSS": "prettier",
+    "TRLC": "prettier",
     "CUE": "cue-fmt",
     "GraphQL": "prettier",
     "HTML": "prettier",

